dts/bindings/rng/nxp,lpc-rng.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2018, Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+description: LPC RNG (True Random Number Generator)
```

Comment:
Drop `True`

Reply:
@galak dropping "True" as it's confusing, the user manual names the RNG device as a True RNG.